ddtrace/internal/forksafe.py

```diff
@@ -55,3 +57,55 @@ def _after_fork():
         ddtrace_after_in_child()
 
     threading._after_fork = _after_fork  # type: ignore[attr-defined]
+
+
+_resetable_objects = weakref.WeakSet()  # type: weakref.WeakSet[ResetObject]
+
+
+def _reset_objects():
+    # type: (...) -> None
+    for obj in _resetable_objects:
+        try:
+            obj._reset_object()
+        except Exception:
+            log.exception("Exception ignored in object reset forksafe hook %r", obj)
+
+
+register(_reset_objects)
+
+
+class ResetObject(wrapt.ObjectProxy):
+    """An object wrapper object that is fork-safe and resets itself after a fork.
+
+    When a Python process forks, a Lock can be in any state, locked or not, by any thread. Since after after fork all
```

Comment:
"after after fork" ?

mistyped and have double "after", or it is meant to be ready as 'after "after fork"' ?

Reply:
mistyped :)